<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ss2_1b_cont_stream_SL</class>
 <widget class="QWidget" name="ss2_1b_cont_stream_SL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="capture_time">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture time[s]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frame_period">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>6.5e-5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="column_en">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0xFF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="row_start">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame period</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Column enable</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Row start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Row end</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="row_end">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>255</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="break_seq">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>120</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Break</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>filename</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename</string>
   </property>
  </widget>
  <widget class="PlotWidget" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>971</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="visualize_data">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>120</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>visualize_data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bits_size">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>60</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bits per pixel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usb_call_init">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Init usb call</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_acq">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>start acquisition</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="debug_mode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>test pattern</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="limited_gate_mode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>limited gate mode</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gate_length">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>1e-8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gate length</string>
   </property>
  </widget>
  <widget class="QComboBox" name="op_mode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>106</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Global shutter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rolling shutter</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="file_path">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>results/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File path</string>
   </property>
  </widget>
  <widget class="QComboBox" name="check_mode">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>150</y>
     <width>106</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>No check</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Headers only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Full check</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Histogram lost frames</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="stop_frame">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>90</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>400000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>90</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop frame</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_ext_trg">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>use external trigger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Projector Patterns Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pattern_line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Projector Frame Time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pattern_time_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the time in seconds. For continuous integration set to -1 .&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
